python-sphinx-gallery/lilac.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+from lilaclib import *
+
+maintainers = [
+        {'github': 'migueldvb', 'email': 'Miguel de Val-Borro <miguel.deval@gmail.com>'}]
+update_on = [{'aur': None}, {'alias': 'python'}]
+repo_depends = ['python-astropy']
```

Comment:
`python-astropy` is not listed in `depends` or `makedepends`.

Reply:
fixed